runtime/nls/j9vm/j9vm.nls

```diff
@@ -1654,6 +1654,20 @@ J9NLS_VM_IDLE_TUNING_PLATFORM_OR_GC_POLICY_NOT_SUPPORTED.system_action=The JVM w
 J9NLS_VM_IDLE_TUNING_PLATFORM_OR_GC_POLICY_NOT_SUPPORTED.user_response=Refer to the documentation for supported platforms and GC policies.
 # END NON-TRANSLATABLE
 
+J9NLS_VM_OPTIONS_MUST_BE_NUMBER=%s must be followed by a number
```

Comment:
Please review the guidance on adding NLS messages at the top of every NLS file:

```
# Note to developers:
#
# New messages MUST be added at the end of this file.
# DO NOT delete messages from this file, as it will change their indices.
# If you wish to remove a message, delete its text, but leave the key in place
```